src/client/csfle/client_encryption.rs

```diff
@@ -182,7 +207,29 @@ impl ClientEncryption {
 
     /// Encrypts a BsonValue with a given key and algorithm.
     /// Returns an encrypted value (BSON binary of subtype 6).
-    pub async fn encrypt(&self, value: bson::RawBson, opts: EncryptOptions) -> Result<Binary> {
+    ///
+    /// To insert or query with an "Indexed" encrypted payload, use a `Client` configured with
+    /// `AutoEncryptionOptions`. `AutoEncryptionOptions.bypass_query_analysis` may be true.
+    /// `AutoEncryptionOptions.bypass_auto_encryption` must be false.
+    pub fn encrypt(
```

Comment:
I think a `#[must_use]` annotation would be useful here so users don't forget to execute their operation. Ditto for any other method that returns an action.

Reply:
Good call, done.